### ISSUE TITLE
Feature: Show detailed console log while doing a remote operation like push

### ISSUE THREAD

**Issue by bendtherules** [maks/MGit#263]
Currently, while doing git push, few overall steps are shown towards the bottom of the UI. It would be nice to have an option to click on it to show expanded details like what shows up on a computer console while doing git push.

**Comment by maks** [comment]
Yes the way all long running operations are handled I'm planning to rework soon, which should allow for this and help with things like #164 too.
